src/loaders/ObjectLoader.js

```diff
@@ -764,12 +764,10 @@ Object.assign( ObjectLoader.prototype, {
 			if ( data.userData !== undefined ) object.userData = data.userData;
 
 			if ( data.children !== undefined ) {
-
-				for ( var child in data.children ) {
-
-					object.add( this.parseObject( data.children[ child ], geometries, materials ) );
-
-				}
+				
```

Comment:
Like @makc said, a classical `for` loop is better here. We are usually not using `forEach`.